### PR TITLE
FIX the values for the column fulfillmentType

### PR DESCRIPTION
**Pull request by alinmigea**
@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: src/Generator/IdealoDE.php [hunk 1]
```diff
@@ -407,20 +410,17 @@ private function row(KeyValue $settings, $items)
             {
                 $data['article_id'] = $this->elasticExportHelper->generateSku($item['id'], self::IDEALO_CHECKOUT, 0, (string)$this->filterAndGetVariationSku($item));
                 $data['itemsInStock'] = $stock;
-                $fulfillmentType = $this->getProperty($item, 'FulfillmentType:Spedition');
+                $data['fulfillmentType'] = '';
 
-                if(!is_null($fulfillmentType) || strlen($fulfillmentType) > 0)
+                if($this->getProperty($item, 'FulfillmentType:Spedition') === true)
```
Comment (Desentyer) [inline src/Generator/IdealoDE.php]: If you define a constant for that value, then use it :D

File: src/Generator/IdealoDE.php [hunk 2]
```diff
@@ -407,20 +410,17 @@ private function row(KeyValue $settings, $items)
...
                 {
-                    $fulfillmentType = 'Spedition';
+                    $data['fulfillmentType'] = 'Spedition';
                 }
-                else
+                elseif ($this->getProperty($item, 'FulfillmentType:Paketdienst') === true)
```
Comment (Desentyer) [inline src/Generator/IdealoDE.php]: same

File: src/Generator/IdealoDE.php (hunk 1 above)
Comment (alinmigea) [inline src/Generator/IdealoDE.php]: yep, true :D I forgot why I defined them #dontdrinkandcode

File: src/Generator/IdealoDE.php (hunk 2 above)
Comment (alinmigea) [inline src/Generator/IdealoDE.php]: modified and I also change another use of the 'checkout approved' constant

**Comment by alinmigea** [comment]
changes moved to fix_for_fulfillment_type_column branch

**Pull request closed without merging**